File: AnimalSaver/Assets/Standard Assets (Mobile)/Scripts/BeeAnimationL.js
#pragma strict

var bumpMap : Texture;
var bumpMap2 : Texture;
var bumpMap3 : Texture;
var start= 0;
var elapsedAnimation=0;
var bee:OTSprite;  
function Start () {
	start=Time.time;
}

function Update () {
	elapsedAnimation++;
	bee = GetComponent(OTSprite);
	
		if(elapsedAnimation%3==0)
		{
			 bee.frameIndex = 1;
		}
		else if(elapsedAnimation%3==1)
		{	
			bee.frameIndex = 2;
		}else if(elapsedAnimation%3==2)
		{	
			bee.frameIndex = 0;
		}


}

File: AnimalSaver/Assets/Standard Assets (Mobile)/Scripts/BeeAnimationL.cs
using UnityEngine;
using System.Collections;

public class BeeAnimationL : MonoBehaviour {
	
	float start=0;
	int elapsedAnimation=0;
	OTSprite bee; 
	// Use this for initialization
	void Start () {
		start=Time.time;
	}
	
	// Update is called once per frame
	void Update () {
		elapsedAnimation++;
		bee = GetComponent<OTSprite>();
	
		if(elapsedAnimation%3==0)
		{
			 bee.frameIndex = 1;
		}
		else if(elapsedAnimation%3==1)
		{	
			bee.frameIndex = 2;
		}else if(elapsedAnimation%3==2)
		{	
			bee.frameIndex = 0;
		}
	}
}
